pypeit/core/extract.py

```diff
@@ -1171,7 +1171,7 @@ def parse_hand_dict(hand_extract_dict):
 
 def objfind(image, thismask, slit_left, slit_righ, inmask = None, FWHM = 3.0,
             hand_extract_dict = None, std_trace = None, ncoeff = 5, nperslit = 10,  bg_smth = 5.0, pkwdth = 3.0,
-            maskwidth = 3.0, sig_thresh = 10.0, peak_thresh = 0.0, abs_thresh = 0.0, trim_edg = (3,3), objmask_nthresh = 2.0,
+            maskwidth = 3.0, sig_thresh = 5.0, peak_thresh = 0.0, abs_thresh = 0.0, trim_edg = (3,3), objmask_nthresh = 2.0,
```

Comment:
So I'm just lagging behind on putting this stuff in the parsets because I've been too busy trying to repair all the things in the code that are broken. All of these hard wired parameters are going to go into the parset.